dbms/src/TiDB/Schema/SchemaBuilder.h

```diff
@@ -29,17 +30,20 @@ struct SchemaBuilder
 
     Context & context;
 
-    std::unordered_map<DB::DatabaseID, TiDB::DBInfoPtr> & databases;
+    KeyspaceDatabaseMap & databases;
 
     Int64 target_version;
 
+    KeyspaceID keyspace_id;
+
     Poco::Logger * log;
 
-    SchemaBuilder(Getter & getter_, Context & context_, std::unordered_map<DB::DatabaseID, TiDB::DBInfoPtr> & dbs_, Int64 version)
+    SchemaBuilder(Getter & getter_, Context & context_, KeyspaceDatabaseMap & dbs_, Int64 version)
         : getter(getter_)
         , context(context_)
         , databases(dbs_)
         , target_version(version)
+        , keyspace_id(getter_.getKeyspaceID())
         , log(&Poco::Logger::get("SchemaBuilder"))
```

Comment:
Better use `log(DB::Logger::get(fmt::format("keyspace={}", keyspace_id)))`, this will add a "[keyspace=222]" in the logging

Reply:
Done.